devel/addon/kyverno/values.yaml

```diff
@@ -0,0 +1,28 @@
+config:
+  # resource types to be skipped by kyverno policy engine
+  # Make sure to surround each entry in quotes so that it doesn't get parsed
+  # as a nested YAML list. These are joined together without spaces in the configmap
+  resourceFilters:
+  - "[Event,*,*]"
+  - "[*,kube-system,*]"
+  - "[*,kube-public,*]"
+  - "[*,kube-node-lease,*]"
+  - "[Node,*,*]"
+  - "[APIService,*,*]"
+  - "[TokenReview,*,*]"
+  - "[SubjectAccessReview,*,*]"
+  - "[SelfSubjectAccessReview,*,*]"
+  - "[*,kyverno,*]"
+  - "[Binding,*,*]"
+  - "[ReplicaSet,*,*]"
+  - "[ReportChangeRequest,*,*]"
+  - "[ClusterReportChangeRequest,*,*]"
+  # Ignore some cert-manager E2E test components that can't easily be configured
+  # to meet the pod security policy.
+  - "[*,bind,*]"
```

Comment:
I didn't find a good way of making bind run non-root, so ignore it instead.
See https://github.com/sameersbn/docker-bind/issues/111 for example.

Reply:
I guess if it wasn't for the acme solver pods, we could have just created a `Policy`/`Policies` for `cert-manager` namespace only and ignored the rest of the cluster- since  I guess we are only testing that `cert-manager` installation is compliant, not all the other software we happen to install for testing?

Reply:
Yes, that would be a better idea. And it'd be less likely that our tests get broken by changes they make to the policies in https://github.com/kyverno/policies